README.md

```diff
@@ -114,31 +114,34 @@ CDTDatastore *datastore = [manager datastoreNamed:@"my_datastore"
                                             error:&outError];
 
 // Create a document
-CDTMutableDocumentRevision *rev = [CDTMutableDocumentRevision revision];
-rev.docId = @"doc1";  // Or don't and get an ID generated for you
-rev.body = @{
+CDTDocumentRevision *rev = [CDTDocumentRevision revisionWithDocId:@"doc1"];
+// Use [CDTDocumentRevision revision] to get an ID generated for you on saving
+rev.body = @[@{
     @"description": @"Buy milk",
     @"completed": @NO,
     @"type": @"com.cloudant.sync.example.task"
-};
+} mutableCopy];
 
 // Add an attachment -- binary data like a JPEG
 CDTUnsavedFileAttachment *att1 = [[CDTUnsavedFileAttachment alloc]
                           initWithPath:@"/path/to/image.jpg"
                           name:@"cute_cat.jpg"
                           type:@"image/jpeg"];
-rev.attachments = @{ att1.name:att1 };
+rev.attachments = @[@{ att1.name:att1 } mutableCopy];
```

Comment:
is the syntax right here, did you mean
```
[@{ att1.name:att1 } mutableCopy];
```
?
Even this looks a bit clunky, is
```
rev.attachments[att1.name] = att1;
```
better?